index.js

```diff
@@ -99,13 +99,18 @@ class Transport { //jshint ignore:line
             opts.title = text;
         }
 
+        var aggregation_key;
+        if (data && data.aggregation_key) aggregation_key = data.aggregation_key;
+
         //efficient way to check if object is empty or error
         if (data instanceof Error) {
             opts.text = text + (text.length ? ' | ' + data.stack : data.stack);
         } else {
             var prop;
-            for (prop in data) {
-                opts.text = text + (text.length ? ' | ' + JSON.stringify(data) : JSON.stringify(data));
+            var dataCopy = Object.assign({}, data);
```

Comment:
Ideally we'd like to avoid having to copy all the data, specifically because of our text property having a possible 4000 characters. In that scenario, servers would experience an unexpected impact.